cmd/kpod/exec.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	execFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "detach, d",
+			Usage: "detached mode: command will run in the background",
```

Comment:
The usage statements some are capitalized to start, some are not.  We should be consistent.   However I just noted that we upper case in some commands and lower case in other kpod command.  @rhatdan these should be all lower to start right?  i.e. "allocate a pseudo TTY".

Reply:
SGTM